app/assets/javascripts/discourse/tests/integration/components/watched-word-uploader-test.js

```diff
@@ -0,0 +1,55 @@
+import componentTest, {
+  setupRenderingTest,
+} from "discourse/tests/helpers/component-test";
+import {
+  createFile,
+  discourseModule,
+} from "discourse/tests/helpers/qunit-helpers";
+import hbs from "htmlbars-inline-precompile";
+import pretender, { response } from "discourse/tests/helpers/create-pretender";
+import { click, waitFor } from "@ember/test-helpers";
+import { setupApplicationTest as EMBER_CLI_ENV } from "ember-qunit";
+
+discourseModule(
+  "Integration | Component | watched-word-uploader",
+  function (hooks) {
+    if (!EMBER_CLI_ENV) {
```

Comment:
Rather than using this you can just use `isLegacyEmber` :)

`import { isLegacyEmber } from "discourse-common/config/environment";`

I would just wrap the whole module in it. We will be able to get rid of all these soon when we finally don't run the legacy tests anymore.